test/Misc/tbi.swift

```diff
@@ -2,10 +2,6 @@
 // RUN:   -O -S %s -parse-as-library -parse-stdlib | \
 // RUN:   %FileCheck --check-prefix=TBI %s
 
-// RUN: %target-build-swift -target arm64-apple-ios7.0 -target-cpu cyclone \
-// RUN:     -O -S %s -parse-as-library -parse-stdlib | \
-// RUN:   %FileCheck --check-prefix=NO_TBI %s
```

Comment:
This is okay for now, but I think it's still worth having *some* kind of reference platform to make sure the `and` instruction gets generated at all.